proguard/1.pro

```diff
@@ -43,17 +29,14 @@
 }
 
 # Legacy Forge config providers
--keep,allowoptimization,allowobfuscation class dev.nolij.zume.** implements *.*.fml.client.IModGuiFactory
+-keep,allowoptimization,allowobfuscation class dev.nolij.zume.** implements *.*.client.gui.ForgeGuiFactory {
+	public <methods>;
+}
 -keepclassmembers,allowoptimization class dev.nolij.zume.** extends *.*.fml.client.config.GuiConfig {
 	public <methods>;
 }
 
 # Fabric entrypoints
 -keep,allowoptimization,allowobfuscation class dev.nolij.zume.FabricZumeBootstrapper
 -keep,allowoptimization,allowobfuscation class dev.nolij.zume.modern.integration.modmenu.ZumeModMenuIntegration
-
--keep @dev.nolij.zumegradle.proguard.ProGuardKeep class * { *; }
--keepclassmembers class * { @dev.nolij.zumegradle.proguard.ProGuardKeep *; }
-
--keep,allowobfuscation @dev.nolij.zumegradle.proguard.ProGuardKeep$WithObfuscation class * { *; }
--keepclassmembers,allowobfuscation class * { @dev.nolij.zumegradle.proguard.ProGuardKeep$WithObfuscation *; }
\ No newline at end of file
+-keep,allowoptimization,allowobfuscation class dev.nolij.zume.primitive.event.KeyBindingRegistrar { public *; }
```

Comment:
`KeyBindingRegistrar` is now gone, and you missed `@ProGuardKeep.WithObfuscation` in `common.pro`